app/services/streaming_exp_award.rb

```diff
@@ -0,0 +1,7 @@
+class StreamingExpAward
+  def perform track
+    if track.uploaded_by && track.length
```

Comment:
This PR is currently blocked by a bug with the tracks getting their length assigned from the id3 tag :(

Reply:
https://github.com/datafruits/streampusher-api/issues/231

Reply:
Should be fixed here https://github.com/datafruits/streampusher-api/commit/422aebaf272e4f17c794c0eb7a1ac1efadd3ddfe